src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,86 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final float GROW_FACTOR = 2f;
+    private Node<K, V>[] table;
+    private int size;
+
+    @SuppressWarnings("unchecked")
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (table.length * LOAD_FACTOR < size) {
+            resize();
+        }
+        int hashKeyNode = hashKey(key);
+        int index = hashKeyNode & (table.length - 1);
```

Comment:
what the purpose of it? you can use hashKeyNode as bucket index. if fact in get method you use hashKey(key) to access bucket